src/FffGcodeWriter.cpp

```diff
@@ -568,6 +576,9 @@ void FffGcodeWriter::processRaft(const SliceDataStorage& storage)
         }
 
         LayerPlan& gcode_layer = *new LayerPlan(storage, layer_nr, z, layer_height, extruder_nr, fan_speed_layer_time_settings_per_extruder_raft_surface, combing_mode, comb_offset, train->getSettingBoolean("travel_avoid_other_parts"), train->getSettingInMicrons("travel_avoid_distance"));
+        // make sure that we are using the correct extruder to print raft
+        gcode_layer.setExtruder(extruder_nr);
```

Comment:
You set the extruder before you set the isinside. This causes a combing bug because the next travel move will assume it started from inside, while the print head is actually at the extruder bay.